manifests/pool6.pp

```diff
@@ -22,8 +23,13 @@
 
   $dhcp_dir = $dhcp::params::dhcp_dir
 
-  concat::fragment { "dhcp_pool_${name}":
+  concat::fragment { "dhcp_pool6_${name}":
```

Comment:
Although this makes sense, it's out-of-scope for this PR. Can you open a new PR for this change?

Reply:
I opened https://github.com/voxpupuli/puppet-dhcp/pull/263 for this, and will remove the section here.